splice_machine/src/main/java/com/splicemachine/derby/vti/SpliceFileVTI.java

```diff
@@ -161,7 +161,10 @@ boolean getPreserveLineEndings(SpliceOperation op) throws StandardException {
                 quotedEmptyIsNull = vtiOperation.getQuotedEmptyIsNull();
             }
 
-            boolean preserveLineEndings = getPreserveLineEndings(op);
+            // if we have oneLineRecords, it doesn't make sense to preserve line endings, since
+            // we don't read the line endings into the data (for this, we would need multiline-line varchars)
+            boolean preserveLineEndings = oneLineRecords ? false : getPreserveLineEndings(op);
+
             if (oneLineRecords && !preserveLineEndings
                     && (charset==null || charset.toLowerCase().equals("utf-8"))) {
```

Comment:
can this be "if (oneLineRecords && && (charset==null || charset.toLowerCase().equals("utf-8")))", ignoring preserveLineEndings ?

Reply:
yes, of course